src/@types/fileObject.ts

```diff
@@ -26,6 +26,19 @@ export interface IpfsFileObject extends BaseFileObject {
   hash: string
 }
 
+export interface S3FileObject extends BaseFileObject {
```

Comment:
what is the purpose of this?
I think we only need
```export interface S3FileObject {
  endpoint: string
  region: string
  objectKey: string
  bucket: string
  accessKeyId: string
  secretAccessKey: string
}```

Reply:
I don't think that we need that.
The files object  (url/s3/ipfs, etc) is encrypted by publisher (using ocean-node's encrypt endpoint) and store in the ddo.

Then, when we need to access the files, we decrypt the string stored in the ddo

Reply:
To clarify this, a DDO is something like:
``
{
  "remote": {
    "type": "s3",
    "s3AccesEncrypted": ".......",
    "encryptedBy": "[node_address]",
    "encryptedMethod": "AES/ECIES"
  }
}
``
and s3AccessEncrypted is an encrypted object with the required params to access s3.
We want to have the object in clarity? and in s3AccesEncrypted to be instead of 0x.... {endpoint:...., region:..., accessKeyId:...}